en/assembly/quick_start_cuav_v5_nano.md

```diff
@@ -0,0 +1,108 @@
+Connect to the flight control GPS interface using a cable.
+
+> **Note**If you use CAN GPS, please use the cable to connect to the flight control CAN interface.
+
+![quickstart](../../assets/flight_controller/cuav_v5_nano/connection/v5_nano_quickstart_04.png)
+
+
+## Safety Switch
+
+The dedicated safety switch that comes with the V5+ is only required if you are not using the recommended Neo GPS (which has an inbuilt safety switch).
+If you are flying without the GPS you must attach the switch directly to the  GPS1  port in order to be able to arm the vehicle and fly (If you use the old 6-pin GPS, please read the definition of the bottom interface to change the line.).
+
+
+## Buzzer
+
+If you do not use the recommended  GPS, there may be problems with the buzzer not working. You may need to add an external buzzer instead.
+
+
+## Radio Control
+
+A remote control (RC) radio system is required if you want to manually control your vehicle (PX4 does not require a radio system for autonomous flight modes).
+You will need to select a compatible transmitter/receiver and then bind them so that they communicate (read the instructions that come with your specific transmitter/receiver).
+
+The figure below shows how you can access your remote receiver (please find the S.Bus cable in the kit)
+
+![quickstart](../../assets/flight_controller/cuav_v5_nano/connection/v5_nano_quickstart_05.png)
+
+
+## Spektrum Satellite Receivers
+
+The V5 nano has a dedicated DSM cable.
+The Spektrum satellite receiver should be connected to the flight control DSM/SBUS/RSSI interface.
+
+
+## Power
+
+The v5 nano kit includes HV\_PM, which supports 2~10s lipo batter. Please connect the 6pin connector of the HW\_PM module to the flight control power1 interface.
+
+![quickstart](../../assets/flight_controller/cuav_v5_nano/connection/v5_nano_quickstart_06.png)
+
+
+## Telemetry System (Optional)
+
+A telemetry system allows you to communicate with, monitor, and control a vehicle in flight from a ground station (for example, you can direct the UAV to a particular position, or upload a new mission).
+
+The communication channel is via Telemetry Radios.
+The vehicle-based radio should be connected to the **TELEM1/TELEM2** port (if connected to this port, no further configuration is required).
+The other radio is connected to your ground station computer or mobile device (usually via USB).
+
+![quickstart](../../assets/flight_controller/cuav_v5_nano/connection/v5_nano_quickstart_07.png)
+
+
+## SD CarD (Optional)
+
+The factory has completed inserting the SD card, you do not need to operate.
+
+
+## Motors
+
+Motors/servos are connected to the MAIN ports in the order specified for your vehicle in the [Airframes Reference](../airframes/airframe_reference.md).
```

Comment:
Note, for nano this is only the MAIN ports. So it won't work for something that uses an AUX port. We should perhaps say that in the introduction.

Reply:
Please help me modify it.